### PR TITLE
Adding bundles for supported versions

### PR DESCRIPTION
**Pull request by jonahjon**
Signed-off-by: jonahjon <jonahjones094@gmail.com>

### CODE REVIEW

File: generatebundlefile/data/input_123_prod.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# This info is hardcoded and comes from https://github.com/aws/eks-anywhere-build-tooling
+name: "v1-21-1001"
```
Comment (ivyostosh) [inline generatebundlefile/data/input_123_prod.yaml:2]:
```suggestion
name: "v1-23-1001"
```

File: generatebundlefile/data/input_120.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# This info is hardcoded and comes from https://github.com/aws/eks-anywhere-build-tooling
+name: "v1-21-1001"
```
Comment (ivyostosh) [inline generatebundlefile/data/input_120.yaml:2]:
```suggestion
name: "v1-20-1001"
```

File: generatebundlefile/data/input_120_prod.yaml [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# This info is hardcoded and comes from https://github.com/aws/eks-anywhere-build-tooling
+name: "v1-21-1001"
```
Comment (ivyostosh) [inline generatebundlefile/data/input_120_prod.yaml:2]:
```suggestion
name: "v1-20-1001"
```

File: generatebundlefile/data/input_123.yaml [hunk 4]
```diff
@@ -0,0 +1,32 @@
+# This info is hardcoded and comes from https://github.com/aws/eks-anywhere-build-tooling
+name: "v1-22-1001"
```
Comment (ivyostosh) [inline generatebundlefile/data/input_123.yaml:2]:
```suggestion
name: "v1-23-1001"
```

File: generatebundlefile/data/input_120.yaml (hunk 2 above)
Comment (junshun) [inline generatebundlefile/data/input_120.yaml:2]: Name should match number

File: generatebundlefile/data/input_120_prod.yaml (hunk 3 above)
Comment (junshun) [inline generatebundlefile/data/input_120_prod.yaml:2]: Same

File: generatebundlefile/data/input_123.yaml (hunk 4 above)
Comment (junshun) [inline generatebundlefile/data/input_123.yaml:2]: Same

**Review by TerryHowe — APPROVED** [review]
/lgtm
/approve

File: generatebundlefile/data/input_120.yaml (hunk 2 above)
Comment (jonahjon) [reply generatebundlefile/data/input_120.yaml]: This name at the moment is irrelevant, but I can change it. The actual bundle name get's imported from the tag of the file in codebuild.

Comment (junshun) [reply generatebundlefile/data/input_120.yaml]: Its fine then you can ignore

**Comment by jonahjon** [comment]
O terry went and fixed them all before I could

**Pull request merged**